compiler/include/astutil.h

```diff
@@ -65,7 +65,8 @@ void reset_ast_loc(BaseAST* destNode, astlocT astloc);
 void reset_ast_loc(BaseAST* destNode, BaseAST* sourceNode);
 
 // compute call sites FnSymbol::calls
-void compute_fn_call_sites(FnSymbol* fn, bool allowVirtual = true);
+void compute_fn_call_sites(FnSymbol* fn);
```

Comment:
We need a comment indicating what is the difference between compute_fn_call_sites and compute_all_fn_call_sites. Also, if compute_fn_call_sites just does the right thing for resolveDynamicDispatches, maybe it could have a better name? like computeCallSitesIgnoringVirtual ?